build.sbt

```diff
@@ -4,12 +4,15 @@ organization := "com.owlcyberdefense"
 
 version := "1.0.0"
```

Comment:
Need to change version number to 1.1.0. This schema is not compatible fully - has new elements in it. 

Not 2.0.0 because only IPv4 checksums are done. Version 2 should compute ALL the various ethernet checksums.